libraries/src/Table/Extension.php

```diff
@@ -112,9 +113,39 @@ public function find($options = array())
 		// Get the DatabaseQuery object
 		$query = $this->_db->getQuery(true);
 
+		// Get the column field types for for the #__extensions Table
+		$tableColumnsType = $this->_db->getTableColumns('#__extensions', false);
+
 		foreach ($options as $col => $val)
 		{
-			$query->where($col . ' = ' . $this->_db->quote($val));
+			if (\in_array($col, (array) $tableColumnsType[$col]))
+			{
+				// Mysql and Postgresql have different properties name for a column data 
```

Comment:
```suggestion
				// Mysql and Postgresql have different properties name for a column data
```
Remove trailing space for making PHPCS pass in drone.